Pulled from http://ftp.vim.org/pub/vim/patches/7.4/7.4.692
Removed the version.c bump.

To: vim_dev@googlegroups.com
Subject: Patch 7.4.692
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.4.692
Problem:    Defining SOLARIS for no good reason. (Danek Duvall)
Solution:   Remove it.
Files:	    src/os_unix.h


*** ../vim-7.4.691/src/os_unix.h	2014-03-25 13:46:22.841832960 +0100
--- src/os_unix.h	2015-04-09 22:04:39.557343160 +0200
***************
*** 65,77 ****
  #endif
  
  /*
-  * Sun defines FILE on SunOS 4.x.x, Solaris has a typedef for FILE
-  */
- #if defined(sun) && !defined(FILE)
- # define SOLARIS
- #endif
- 
- /*
   * Using getcwd() is preferred, because it checks for a buffer overflow.
   * Don't use getcwd() on systems do use system("sh -c pwd").  There is an
   * autoconf check for this.
--- 65,70 ----
